app/design/frontend/base/default/template/boltpay/replace.phtml

```diff
@@ -25,7 +25,10 @@
 if(!$this->isAllowedReplaceScriptOnCurrentPage()
    || !$this->canUseBolt()
    || !$this->getPublishableKeyForRoute()
-) return;
+    || Mage::app()->getRequest()->getRouteName() === 'catalog' && Mage::app()->getRequest()->getControllerName() === 'product'
```

Comment:
why? can't we use both minicart and product page checkout?

Reply:
We have two separate options. Enable Everywhere and enable product checkout.
So this condition will work when "Everywhere" disabled.

Reply:
I'm confused. This condition is used to decide to exist condition. 
so, even if `$this->isAllowedReplaceScriptOnCurrentPage()` returns true, current logic will not execute replace.phtml if user is in product page, which seems wrong